guides/common/modules/proc_installing-packages-on-a-host.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host.
```

Comment:
I'd replace this line with something like:
`You can review and install packages on a host using the {WebUI}.` -- sorry, I don't remember the variable name exactly :D

Reply:
`{ProjectWebUI}`: see https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L84